src/_posts/2020-02-18-solve-your-self-proclaimed-stowaway.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+asset-type: post
+name: solution-to-a-self-proclaimed-stowaway
+title: Solution to an apprentice's self proclaimed Imposter Syndrome
+date: 2020-02-18 00:00:00 +00:00
+author: José Pablo Wenzel
+image:
+    src: /assets/custom/img/blog/2020-02-18-solve-your-self-proclaimed-stowaway/imposter-landscape.jpg
+tags:
+    - life at codurance
+---
+
+# How to go through the apprenticeship program without feeling like an imposter
```

Comment:
I know you talk from a personal perspective, but this is a topic that could be extensive to a whole variety of expertise and seniority levels, I mean, it not only applies to apprentices so you can even generalise the topic, as is a wide spread sympton

Reply:
I understand, but I've done the post aiming for future apprentices, I understand that the imposter syndrome is quite wide, and I have so much to talk about it, but I wanted to make it concise to the context of the program, hope it makes sense